packit/command_handler.py

```diff
@@ -110,6 +110,26 @@ def run_command(
 class SandcastleCommandHandler(CommandHandler):
     name = RunCommandType.sandcastle
 
+    def __init__(self, local_project: LocalProject, config: Config):
+        super().__init__(local_project=local_project, config=config)
+        self.local_project = local_project
+        self.config = config
+
+        # we import here so that packit does not depend on sandcastle (and thus python-kube)
+        from sandcastle.api import Sandcastle, MappedDir
+
+        self.mapped_dir = MappedDir(
+            local_dir=self.local_project.working_dir,
+            path=self.config.command_handler_work_dir,
+            with_interim_pvc=True,
+        )
+        self.sandcastle = Sandcastle(
+            image_reference=self.config.command_handler_image_reference,
+            k8s_namespace_name=self.config.command_handler_k8s_namespace,
+            mapped_dir=self.mapped_dir,
+        )
+        self.sandcastle.run()
```

Comment:
Would it make sense to move this in a separate method to be called from `run_command()` in case `self.sandcastle` is `None`? AFAIK, it's a good practice to keep `__init__`s as lightweight as possible.

Reply:
yes, I can make it a lazy property - since CommandHandler is initiated lazily in Upstream/DistGit I didn't bother with that here